pilot/pkg/serviceregistry/kube/controller/multicluster.go

```diff
@@ -50,7 +52,7 @@ type Multicluster struct {
 
 // NewMulticluster initializes data structure to store multicluster information
 // It also starts the secret controller
-func NewMulticluster(kc kubernetes.Interface, secretNamespace string, opts Options,
+func NewMulticluster(kc kubernetes.Interface, secretNamespace string, opts Options, caroot string,
```

Comment:
We have Options object with all kind of options - can we put the caroot there as well ?

Reply:
put it into Options